docs/asciidoc/front_matter.adoc

```diff
@@ -1,8 +1,13 @@
-== Introduction
+////
+TODO:
+* I need to clear up the section showing the GET requests, clarify the "ID", and also probably mention these API calls are examples and we make not guarantees that identifiers.org/n2t.net won't change.
+* do we want to add a self_url to the JSON response from DRS objects?  This would make it much, much easier to construct the access method rather than using the hostname-based URI which requires manipulation and parsing.
```

Comment:
Yeah, I think that could be useful. At least as an optional field so DRS servers can opt out if they want to.

Reply:
yep, I'll file a bug for this one so we don't loose track of the idea for a future version

Reply:
See https://github.com/ga4gh/data-repository-service-schemas/issues/320

Reply:
Now that the bug is filed, please remove this comment from the spec.